debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java

```diff
@@ -218,6 +226,15 @@ public String getAlias() {
                     " in case something else is processed this transform can log it as warning, error or stop the" +
                     " process");
 
+    static final Field OPERATION_DELETE_BEHAVIOR = Field.create("debezium.op.delete.enabled")
```

Comment:
Is this opt-in flag really needed? Couldn't this just be based on whether the field identified by "table.field.event.is_deleted" is present or not?

Reply:
Yeah that's a good point, I had thought about that but wasn't sure which way would be best. I'll remove that!